help/de/Help_ch_Account-Actions.xml

```diff
@@ -708,83 +694,107 @@
-
-                  <listitem>
-                    <para>Wechseln Sie in dieses Verzeichnis, öffnen Sie eine Root-Shell und führen Sie den Befehl
-                      <code>gnc-fq-update</code> aus. Dadurch wird eine Perl CPAN-Update-Sitzung
-                      gestartet, die ins Internet geht und das Modul
-                      <application>Finance::Quote</application> und seine Abhängigkeiten auf Ihrem
-                      System installiert. Das Programm gnc-fq-update ist interaktiv, bei den meisten
-                      Systemen sollten Sie jedoch die erste Frage mit <quote>Nein</quote>
-                      beantworten können: <quote>Sind Sie bereit für die manuelle Konfiguration?
-                      [ja]</quote> und die Aktualisierung wird automatisch fortgesetzt von diesem
-                      Punkt an.
-                    </para>
-                  </listitem>
-                </orderedlist>
-              </listitem>
-            </itemizedlist>
+                <listitem>
+                  <para>Trifft keine der obigen Anleitungen auf Ihre Umgebung zu, so können Sie die folgende alternative,
+                    allgemeinere Anweisungen probiern:
+                  </para>
 
-            <para>Nachdem die Installation abgeschlossen ist, sollten Sie das Testprogramm <code>gnc-fq-check</code>,
-              das mit &app; verteilt wird, im gleichen Verzeichnis ausführen, um zu testen, ob
-              <application>Finance::Quote</application> installiert ist und richtig funktioniert.
+                  <orderedlist spacing="compact">
+                    <listitem>
+                      <para>Schließen Sie alle &app;-Anwendungen, die Sie gerade ausführen.
+                      </para>
+                    </listitem>
+
+                    <listitem>
+                      <para>Lokalisieren Sie den Ordner, in dem &app; installiert ist, indem Sie nach
+                        <code language="shell">gnc-fq-update</code> suchen.
+                      </para>
+                    </listitem>
+
+                    <listitem>
+                      <para>Wechseln Sie in dieses Verzeichnis, öffnen Sie eine Root-Shell und führen Sie den Befehl
+                        <code language="shell">gnc-fq-update</code> aus. Dadurch wird eine Perl
+                        CPAN-Update-Sitzung gestartet, die ins Internet geht und das Modul
+                        <application>Finance::Quote</application> und seine Abhängigkeiten auf
+                        Ihrem System installiert. Das Programm
+                        <code language="shell">gnc-fq-update</code> ist interaktiv, bei den meisten
+                        Systemen sollten Sie jedoch die erste Frage mit <quote>Ja</quote>
+                        beantworten können: <quote>Sind Sie bereit für die automatische
+                        Konfiguration? [ja]</quote> und die Aktualisierung wird von diesem Punkt an
+                        selbstständig ablaufen.
```

Comment:
Das ist die CPAN-Konfiguration.

Falls das nicht bereits im Guide-Glossar alles definiert wurde:
https://wiki.gnucash.org/wiki/Online_Quotes#Installing_Finance::Quote_using_CPAN,
https://wiki.gnucash.org/wiki/CPAN

Reply:
Jetzt stellt sich mir die Frage, wo die Anleitung der Installation besser aufgehoben ist, in der Hilfe oder im Guide? Ich halte die Hilfe für den richtigen Ort und dann soltle der Guide umgebaut werden. Also den Abschnitt dort kürzen und auf die Hilfe verlinken.

Reply:
@fellen: Wie ist deine Meinung? Beschreibung in Help oder Guide?

Reply:
> Ich halte die Hilfe für den richtigen Ort und dann soltle der Guide umgebaut werden. Also den Abschnitt dort kürzen und auf die Hilfe verlinken.

Sagt mein Gefühl auch. Technische Vorraussetzungen gehören  in die Hilfe.
Ich habe mal ähnliche "Bugs" gesucht, die Gedanken zum Pro&Con enthlten:
[Large portions of 2.9 Transactions should be moved to the Help document](https://bugs.gnucash.org/show_bug.cgi?id=797364)
Grundlegende Gedanken:[Proposal to restructure Tutorial and Concepts Guide](https://bugs.gnucash.org/show_bug.cgi?id=687820) aber mit Fehlern;
Gegenrichtung: [Move contents of Help Chapter 3. Getting Started into Guide Chapter 2 The Basics Section 3 Interface](https://bugs.gnucash.org/show_bug.cgi?id=796855), fixed

Auch zum Thema F::Q: [Linux section about Finance::Quote Installation misleading](https://bugs.gnucash.org/show_bug.cgi?id=797974)

Ich bin etwas hin und her gerissen, wo man das am Besten unterbringt.
- In Tools and Assistants gibt's das Online Banking Setup, ist ja im Menü unter Tools.
- In Customizing Gnucash gibt es bereits Changing the Language or Region
- Anhang A. GnuCashTips und Tricks hat viel zu F::Q (aber im Anhang möchte ich eigentlich nur solche Tabellen.
- oder ein neues Kapitel? Gibt es noch was dafür?

Reply:
Wenn ich mir das so anschaue komme ich zu folgender Meinung bezüglich der Installationsanleitung von F::Q :

1. Der aktuelle Platz im Abschnitt `Konto erstellen` ist falsch. Ich will ein Konto anlegen und kein Perl-Modul installieren.
2. Bei `Tools and Assistants` kann die Installationsbeschreibung im Abschnitt `Price Editor - Configuration...` eingefügt werden.
3. Verknüpfung in `Konto erstellen` auf `Tools and Assistants` einfügen, damit der Anwender weis, was zu tun ist um die Kursabfrage ans laufen zu bekommen.

Reply:
Ja, bildet ein Thema mit Kursdatenbank und Wertpapier-Editor, aber noch eigentlicher mit
Customizing GnuCash:
Internal Settings: bisherige 10.1-6
External Configuration
  Sprache
  F::Q
  MRU (Einzige gsettings Option ohne Dialog)

Reply:
Ah, jetzt habe ich die Kapitelnummern gefunden, du bist im HTML unterwegs. Ich schaue immer nur bei Yelp und da gibt es keine Nummern.

Ich kann die Integration bei "Customizing" unterstützen! Was meinst du mit "External Configuration" und "MRU"?

Reply:
Konfiguration außerhalb der Programmoberfläche.

https://wiki.gnucash.org/wiki/FAQ#Q:_How_can_I_change_the_number_of_the_most_recent_files_.28MRU.29_list.3F

Reply:
Ich habe jetzt die Informationen rund um die Installation von F::Q auskommentiert.

Oder soll ich das sofort entfernen? Aber das könnte den Backport unnötig erschweren.